plugins/package-managers/spdx/src/funTest/assets/projects/synthetic/subproject-conan/project-xyz.spdx.yml

```diff
@@ -16,7 +16,7 @@ externalDocumentRefs:
     spdxDocument: "./subproject/subproject-with-conan-reference.spdx.yml"
     checksum:
       algorithm: "SHA1"
-      checksumValue: "53840294281f7d1bf401b746162da11dcf9306eb"
+      checksumValue: "032a087b0da8f353c03671a72179b807b4ec7eb3"
```

Comment:
I believe we have test for `SpdxResolvedDocument` in place that verify the checksum, so please explain in the commit message how this could have been wrong.

Reply:
I tried to explain it.

Reply:
Ok, thanks. Let me try to rephrase that for my understanding: The checksum was always wrong, but that fact did not result in an issue or test failure until the previous commit. Now that issues are properly propagated, it would result in a test failure of `SpdxDocumentFileFunTest` whose expected result files contain no issues. Correct?

Reply:
Correct.